models/staging/bigquery_audit_log/stg_bigquery_audit_log__data_access.sql

```diff
@@ -1,7 +1,20 @@
-with source as (
+-- if you change anything in this file, please copy everything from line 11 on (`source CTE` -> EOF)
+-- into `stg_bigquery_audit_log__data_access_query_text.txt` in this same folder. If you don't and run
+-- `leaner_query_setup.py`, you will not see the changes to this staging file in your model.
+
+with unioned as (
+    {{ union_relations(
```

Comment:
This is `union_realtions`, but the python script adds it as `dbt_utils.union_relations`. Is that intentional?

Reply:
Not intentional - fixed.